README.md

```diff
@@ -4,24 +4,26 @@
 <a href="https://github.com/camaraproject/§repo_name§/graphs/contributors" title="Contributors"><img src="https://img.shields.io/github/contributors/camaraproject/§repo_name§?style=plastic"></a>
 <a href="https://github.com/camaraproject/§repo_name§" title="Repo Size"><img src="https://img.shields.io/github/repo-size/camaraproject/§repo_name§?style=plastic"></a>
 <a href="https://github.com/camaraproject/§repo_name§/blob/main/LICENSE" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
+<a href="https://github.com/camaraproject/§repo_name§/releases/latest" title="Latest Release"><img src="https://img.shields.io/github/release/camaraproject/§repo_name§?style=plastic"></a>
 
 # §repo_name§
-Repository to describe, develop, document and test the §repo_name§ API family
+Repository to describe, develop, document, and test the §repo_name§ API family
 
 ## Scope
 * Service APIs for “§repo_name§” (see APIBacklog.md)  
 * It provides the customer with the ability to:  
   * §API family description§.
-  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
-* Describe, develop, document and test the APIs (with 1-2 Telcos)  
+  * NOTE: The scope of this API family should be limited (at least at the first stage) to 4G and 5G.  
+* Describe, develop, document, and test the APIs (with 1-2 Telcos)  
 * Started: §start date§
-* Location: virtually  
 
-## Meetings
-* Meetings are held virtually
-* Schedule: tbd
-* Meeting link: tbd
-
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-§api_family_abbreviation§>.
-* A message to all Contributors of this Sub Project can be sent using <sp-§api_family_abbreviation§@lists.camaraproject.org>.
+## Contributing
+* Meetings
+    * [Registration](https://wiki.camaraproject.org/x/TQAG) Update this link with your meeting join link
+    * Schedule: 
+    * Minutes: Access [meeting minutes] (update this link to your wiki page using the "Share" link).
+* Mailing List
+    * Subscribe / Unsubscribe to the mailing list of this Sub Project <https://lists.camaraproject.org/g/sp-§api_family_abbreviation§>.
+    * A message to all Contributors of this Sub Project can be sent using <sp-§api_family_abbreviation§@lists.camaraproject.org>.
+* [Latest Release](https://github.com/camaraproject/§repo_name§/releases/latest)
+    * Release Tracking: Follow updates on our [wiki release tracking page] (Link to wiki release tracking page using the "share" link from Confluence)
```

Comment:
The release tracking within wiki is meant for release planning and management. The source of truth about existing release should be here in GitHub within CHANGELOG.md and the release notes of GitHub (which are a copy of the CHANGELOG.md content).